REM-1307: Get particular batch status by identifier (#15)
Remmeauth/remme-core-cli: README.md

368440.0
```

### Batches

Get batch status by its identifier — ``remme batch get-status``:

| Arguments | Type   |  Required | Description                                         |
| :-------: | :----: | :-------: | --------------------------------------------------- |

> | id        | String |  Yes      | Get batch status by its identifier.                                |

Reviewer: ```suggestion
| id        | String |  Yes      | Identifier to get a batch status by.                                |
```

Author: Done.

| node-url  | String |  No       | Apply the command to the specified node by its URL. |

```bash
$ remme batch get-status \
      --id=ce5d2047f1a34bafbe228fac392573435b7048352b1f013af69e6bbf8fc10e5156915d7ee94f0d4a2fe363e7583f7039f29b1ab50d2e139e0957b5d08740bfd9 \
      --node-url=node-genesis-testnet.remme.io
```

> 

Reviewer: Remove blank line.

Author: Done.

}
```

### Batch

> Get batch status by identifier — ``remme batch get-status``:

Reviewer: ```suggestion
Get a batch status by its identifier — ``remme batch get-status``:
```

Author: Done.

| Arguments   | Type   |  Required | Description                       |
| :---------: | :----: | :-------: | --------------------------------  |

> | id          | String |  Yes      | Identifier to get batch by. |

Reviewer: It is necessary that nothing climbs inside the table.

Author: Done.

}
```

Get a batch status by its identifier — ``remme batch get-status``:

| Arguments   | Type   |  Required | Description                       |
| :---------: | :----: | :-------: | --------------------------------  |

> | id          | String |  Yes      | Identifier to get batch by.       |

Reviewer 2: `Identifier to get a batch status by.`

Author: Done.

}
```

Get a batch status by its identifier — ``remme batch get-status``:

| Arguments   | Type   |  Required | Description                       |

> | :---------: | :----: | :-------: | --------------------------------  |

Reviewer: ```suggestion
| :-------: | :----: | :-------: | --------------------------------  |
```

Author: Done.

> | id          | String |  Yes      | Identifier to get a batch status by.       |

Reviewer: It is necessary that nothing climbs outside the table layout.

Author: Done.

Reviewer: ```suggestion
| id          | String |  Yes      | Identifier to get a batch status by. |
```
